applications: asset_tracker_v2: Power optimize LTE usage (#13851)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

  * The :kconfig:option:`NRF_CLOUD_SEND_SERVICE_INFO_FOTA` and :kconfig:option:`NRF_CLOUD_SEND_SERVICE_INFO_UI` Kconfig options.
    The application no longer sends a device shadow update; this is now handled by the :ref:`lib_nrf_cloud` library.

* Updated:

>   * Few power optimizations to LwM2M configuration overlay:

Reviewer: ```suggestion
  * The following power optimizations to the LwM2M configuration overlay:
```

    * Enable DTLS Connection Identifier.

>     * Do LwM2M update once a hour and try to request similar periodic tracking area update intervall from LTE network.

Reviewer: ```suggestion
    * Perform LwM2M update once an hour and request for similar update interval of periodic tracking area from the LTE network.
```
I may have messed up this big time. However this needs some attention. The long list of substantives "tracking area update interval" bothered me a bit.

>     * Request same active time as our QUEUE mode polling time is.

Reviewer: ```suggestion
    * Request the same active time as the QUEUE mode polling time.
```

>     * Enable eDRX with smallest interval and short paging window.

Reviewer: ```suggestion
    * Enable eDRX with shortest possible interval and a short paging window.
```

>     * Enable LwM2M engine's tickless mode.

Reviewer: ```suggestion
    * Enable tickless mode in the LwM2M engine.
```
